src/EventStore.ClientAPI/IEventStoreConnection.cs

```diff
@@ -338,6 +338,33 @@ public interface IEventStoreConnection : IDisposable
             int bufferSize = 10,
             bool autoAck = true);
 
+        /// <summary>
+        /// Asynchronously subscribes a persistent subscription (competing consumer) to the event store
```

Comment:
Would it be better if we instead of `Asynchronously subscribes a persistent subscription (competing consumer) to the event store` say `Asynchronously subscribes to a persistent subscription (competing consumer) to event store`?